### PR TITLE
Jquery

### PR DESCRIPTION
**Pull request by allopez7**

### CODE REVIEW

**Comment by songz** [comment]
looks good! Please merge in meme branch first please.

**Review by songz — APPROVED** [review]

File: Meme/public/index.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+    const button = document.getElementById('submit');
+    button.onclick = () =>{
+      const name = document.getElementById('name').value;
+      const comment = document.getElementById('comment').value;
+      const httpReq = new XMLHttpRequest();
+      httpReq.open('POST','/send');
+      httpReq.setRequestHeader('content-type','application/json');
+      httpReq.send(
+        JSON.stringify({
```
Comment (songz) [inline Meme/public/index.html]: `{name, comment}`

File: Meme/express.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+app.listen(3608);
+app.use(express.static('public'));
+app.use(bp.json({limit: "500mb"}));
+
+const response = (req,res) =>{
+  fs.appendFile('/home/alopez/garagescript/Meme/public/book.txt',`${req.body.name}: ${req.body.comment}`+'\n',()=>{});
+  res.send();
+}
+app.post('/send', response);
```
Comment (saumya665) [inline Meme/express.js]: you can just combine the app.post and response into one function

File: Meme/express.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+const resp2 = (req, res) =>{
+  fs.writeFile('/home/alopez/garagescript/Meme/public/'+req.body.name+'.'+'png',req.body.Canvas.replace('data:image/png;base64',""),'base64',() =>{
+   gm('/home/alopez/garagescript/Meme/public/'+req.body.name+'.'+'png').fontSize(40).drawText(50,50,req.body.comment).write('/home/alopez/garagescript/Meme/public/text.png',(err)=>{
+     console.log("error is", err);
+   });
+});
+  res.send('done');  
+}
+app.post('/image', resp2);
```
Comment (saumya665) [inline Meme/express.js]: might be simpler to include resp2 in app.post

File: Meme/public/index.html [hunk 4]
```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title></title>
```
Comment (saumya665) [inline Meme/public/index.html]: nothing in title?

File: Meme/public/index.html [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+  httpReq2.onreadystatechange = ()=>{
+    if(httpReq2.responseText){
+      display.innerHTML = httpReq2.responseText;
+    }
+  }
+
+  httpReq2.send();
+  
+} 
+
```
Comment (saumya665) [inline Meme/public/index.html]: change the indentation and spacing

File: jquery/public/app.js [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+    url: '/send',
+    contentType: 'application/json',
+    data: JSON.stringify(
+      {name: name,comment: comment}
+    ),
+    success: (data)=>{
+      console.log('Success Text: ', data);
+    }
+  });
+
```
Comment (saumya665) [inline jquery/public/app.js:16]: extra space here

File: jquery/public/app.js [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+    display.html(data);
+    }
+  );
+}
+
+window.setInterval(getTextFile, 1000);
+
+//video 
+const video = $('#video')[0];
+
```
Comment (saumya665) [inline jquery/public/app.js]: extra space

File: jquery/public/index.html [hunk 8]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <script src="https://code.jquery.com/jquery-3.2.1.js"></script> 
+  <meta charset="UTF-8">
+  <title></title>
```
Comment (saumya665) [inline jquery/public/index.html]: empty title

**Review by saumya665 — APPROVED** [review]

**Pull request merged**